packages/server/src/tests/banners/bannerSelection.ts

```diff
@@ -189,9 +189,8 @@ export const selectBannerTest = (
     }
 
     for (const test of tests) {
-        const deploySchedule = enableScheduledDeploys
-            ? targetingTest?.deploySchedule ?? defaultDeploySchedule
-            : undefined;
+        // eslint-disable-next-line prettier/prettier
```

Comment:
why did this file change?

Reply:
Because of this error in GA:

![image](https://github.com/user-attachments/assets/b8ee53f8-09ee-4e63-8106-0568796b74f8)

I tried updating the line as requested, but that caused another error in my IDE, so I've just ignored the line